src/app/modules/terminal/components/terminal/terminal.component.ts

```diff
@@ -65,6 +73,26 @@ export class TerminalComponent implements OnInit, AfterViewInit, OnDestroy {
         this.reconnect();
       });
     }
+
+    this.store$.pipe(
+      waitForPreferences,
+      filter((preferences) => Boolean(preferences.sidenavStatus)),
+      untilDestroyed(this),
+    ).subscribe(() => {
+      if (this.shellConnected) {
+        // 300ms waiting for sidenav width to be set
```

Comment:
Better to store `300` in a properly named constant in the class. The variable name should be explanation of the value instead of a comment.

Reply:
ok :thumbsup: